go/vt/vtctl/vtctl.go

```diff
@@ -1576,6 +1602,29 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 		wr.Logger().Infof("keyspace %v already exists (ignoring error with -force)", keyspace)
 		err = nil
 	}
+	if err != nil {
+		return err
+	}
+	// create empty vschema.Keyspace if it doesn't exist yet
+	_, err = wr.TopoServer().GetVSchema(ctx, keyspace)
+	if err != nil {
+		wr.Logger().Infof("error from GetVSchema %v", err)
+		if topo.IsErrType(err, topo.NoNode) {
+			vs := &vschemapb.Keyspace{
+				Tables: map[string]*vschemapb.Table{
+					"unsharded": {},
+				},
+				KeyspaceType: ktype,
+			}
```

Comment:
@sougou does this look correct for the default vschema? Unless I create at least one entry in the map, it gets thrown out as empty vschema and nothing gets created.